feature-push-notifications/src/main/java/io/novafoundation/nova/feature_push_notifications/data/data/settings/PushSettingsSerializer.kt

```diff
@@ -0,0 +1,11 @@
+package io.novafoundation.nova.feature_push_notifications.data.data.settings
+
+import com.google.gson.GsonBuilder
+import io.novafoundation.nova.common.utils.gson.SealedTypeAdapterFactory
+
+object PushSettingsSerializer {
+
+    fun gson() = GsonBuilder()
+        .registerTypeAdapterFactory(SealedTypeAdapterFactory.of(PushSettings.ChainFeature::class))
+        .create()
```

Comment:
Can we specify Type adapter for a field? To avoid creating separate `gson` instance